rcl_logging_spdlog/README.md

```diff
@@ -2,6 +2,16 @@
 
 Package supporting an implementation of logging functionality using `spdlog`.
 
+The `rcl_logging` interface allows [rcl_logging_spdlog/logging_interface.h](include/rcl_logging_spdlog/logging_interface.h):
+ - initialize
+ - log a message
+ - set the logger level
+ - shutdown
+
+Further still there are some useful abstractions and utilities:
+- Macros for controlling symbol visibility on the library
+ - [rcl_logging_spdlog/visibility_control.h](include/rcl_logging_spdlog/visibility_control.h)
```

Comment:
@ahcorde AFAIK these are for internal use only. Pretty much every package has a visibility control header.

Reply:
other packages such as rclcpp consider this a [feature](http://docs.ros2.org/eloquent/api/rclcpp/index.html)

Reply:
It documents them as internal utilities (!= feature IMHO).